spec.html

```diff
@@ -37913,7 +37740,7 @@ <h1>Promise Resolve Functions</h1>
             1. Let _thenAction_ be _then_.[[Value]].
             1. If IsCallable(_thenAction_) is *false*, then
               1. Return FulfillPromise(_promise_, _resolution_).
-            1. Perform EnqueueJob(`"PromiseJobs"`, PromiseResolveThenableJob, &laquo; _promise_, _resolution_, _thenAction_ &raquo;).
+            1. Perform HostEnqueuePromiseJob(PromiseResolveThenableJob, &laquo; _promise_, _resolution_, _thenAction_ &raquo;).
```

Comment:
Can this produce an abrupt completion? If so, let’s prefix it with ?; if not, ! (same throughout)

Reply:
I would be happy to add ! here; any concerns?

Reply:
Then we should also add a sentence to the specification of HostEnqueuePromiseJob, like "An implementation of HostEnqueuePromiseJob must complete normally in all cases." (copied from [HostReportErrors](https://tc39.es/ecma262/#sec-host-report-errors))